migrations/v4.js

```diff
@@ -0,0 +1,66 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v3.0.0 > v4.3.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v3.0.0', { name: 'adapt-contrib-assessment', version: '<4.3.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _assessment }) => _assessment !== undefined);
+    return assessments.length;
+  });
+
+  mutateContent('adapt-contrib-assessment - add assessment._scrollToOnReset', async () => {
+    assessments.forEach(assessment => {
+      assessment._scrollToOnReset = false;
+    });
```

Comment:
This is adding _scrollToOnReset to the article rather than assessment
```suggestion
    assessments.forEach(({ _assessment }) => {
      _assessment._scrollToOnReset = false;
    });
```